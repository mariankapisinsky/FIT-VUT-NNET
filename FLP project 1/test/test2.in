S,A,B,C
a,b,c
S
S->ASBS
S->BSCS
S->ASCS
A->Ab
A->Ac
A->a
B->aB
B->b
C->aAC
C->c
