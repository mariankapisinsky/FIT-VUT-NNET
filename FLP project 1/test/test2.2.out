S,<SBS>,<BS>,<SCS>,<CS>,A,b',c',B,a',C,<AC>
a,b,c
S
S->A<SBS>
<SBS>->S<BS>
<BS>->BS
S->B<SCS>
<SCS>->S<CS>
<CS>->CS
S->A<SCS>
A->Ab'
b'->b
A->Ac'
c'->c
A->a
B->a'B
a'->a
B->b
C->a'<AC>
<AC>->AC
C->c
